CMakeLists.txt

```diff
@@ -96,10 +96,13 @@ elseif( CMAKE_CXX_COMPILER MATCHES ".*/hcc$" )
   message( STATUS "HCC compiler set; ROCm backend selected [ CXX=/opt/rocm/bin/hcc cmake ... ]" )
 endif( )
 
+# NOTE:  workaround until hcc & hip cmake modules fixes symlink logic in their config files; remove when fixed
+list( APPEND CMAKE_PREFIX_PATH ${ROCM_PATH} ${ROCM_PATH}/hcc ${ROCM_PATH}/hip )
```

Comment:
This should be `list( APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/hcc /opt/rocm/hip )`. This is for users building library. However, when building the packages for release it can be set to `-DCMAKE_PREFIX_PATH=/opt/rocm-<ver>;/opt/rocm-<ver>/hcc;/opt/rocm-<ver>/hip`, which it will search in `/opt/rocm-<ver>` first before searching in `/opt/rocm`.